Resources/Prototypes/Entities/Mobs/NPCs/Rslimes.yml

```diff
@@ -0,0 +1,308 @@
+        color: "#00ff5f"
+
+- type: entity
+  id: ReagentSlimeLead
+  parent: ReagentSlime
+  suffix: Lead
+  components:
+  - type: Bloodstream
+    bloodReagent: Lead
+  - type: PointLight
+    color: "#5C6274"
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Aliens/elemental.rsi
+    layers:
+      - map: [ "enum.DamageStateVisualLayers.Base" ]
+        state: alive
+        color: "#5C6274"
+
+- type: entity
+  id: ReagentSlimechlorinetriflouride
+  parent: ReagentSlime
+  suffix: chlorinetriflouride
+  components:
+  - type: Bloodstream
+    bloodReagent: ChlorineTrifluoride
+  - type: PointLight
+    color: "#FFC8C8"
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Aliens/elemental.rsi
+    layers:
+      - map: [ "enum.DamageStateVisualLayers.Base" ]
+        state: alive
+        color: "#FFC8C8"
+
+- type: entity
+  id: ReagentSlimePotassium
+  parent: ReagentSlime
+  suffix: Potassium
+  components:
+  - type: Bloodstream
+    bloodReagent: Potassium
+  - type: PointLight
+    color: "#c6c8cc"
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Aliens/elemental.rsi
+    layers:
+      - map: [ "enum.DamageStateVisualLayers.Base" ]
+        state: alive
+        color: "#c6c8cc"
+        
+- type: entity
+  id: reagentslimeVents
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    startAnnouncement: true
+    earliestStart: 20
+    reoccurrenceDelay: 12
+    minimumPlayers: 1
+    weight: 6 # Really weak compared to other critters
+    duration: 60
+  - type: VentCrittersRule
+    entries:
+    - id: ReagentSlime
+      prob: 0.01
+    - id: ReagentSlimeBeer
+      prob: 0.01
+    - id: ReagentSlimePax
+      prob: 0.01
+    - id: ReagentSlimeNocturine
+      prob: 0.01
+    - id: ReagentSlimeTHC
+      prob: 0.01
+    - id: ReagentSlimeBicaridine
+      prob: 0.01
+    - id: ReagentSlimeToxin
+      prob: 0.01
+    - id: ReagentSlimeNapalm
+      prob: 0.01
+    - id: ReagentSlimeOmnizine
+      prob: 0.01
+    - id: ReagentSlimeMuteToxin
+      prob: 0.01
+    - id: ReagentSlimeNorepinephricAcid
+      prob: 0.01
+    - id: ReagentSlimeEphedrine
+      prob: 0.01
+    - id: ReagentSlimeRobustHarvest
+      prob: 0.01
+    - id: ReagentSlimeIchor
+      prob: 0.01
+    - id: ReagentSlimeBleach
+      prob: 0.01
+    - id: ReagentSlimeSoap
+      prob: 0.01
+    - id: ReagentSlimeSpacelube
+      prob: 0.01
+    - id: ReagentSlimeBuzzachloricbees
+      prob: 0.01
+    - id: ReagentSlimeWehjuice
+      prob: 0.01
+    - id: ReagentCognizine
+      prob: 0.01
+    - id: ReagentSlimeOpporozidone
+      prob: 0.01
+    - id: ReagentSlimeNecrosol
+      prob: 0.01
+    - id: ReagentSlimeSpaceDrugs
+      prob: 0.01
+    - id: ReagentSlimeUnstableMutagen
+      prob: 0.01
+    - id: ReagentSlimeLead
+      prob: 0.01
+    - id: ReagentSlimechlorinetriflouride
+      prob: 0.01
+    - id: ReagentSlimePotassium
+      prob: 0.01
+    - id: ReagentSlimeLotophagoiOil
+      prob: 0.01
```

Comment:
Im not gona request Changes but not gona approve this, i was against this on floof, will be here.

Those are only for anomalies.